app/assemblers/assemble_track_switcher.rb

```diff
@@ -0,0 +1,28 @@
+class AssembleTrackSwitcher
```

Comment:
This is for assembling the track switcher parameters.

Reply:
This would be better as a serializer, as it doesn't do any filtering or searching. So I've changed it to be `SerializeTracksForDropdown`.